packages/mask/src/plugins/NextID/components/BindDialog.tsx

```diff
@@ -17,12 +17,12 @@ import { bindProof } from '@masknet/web3-providers'
 interface BindDialogProps {
     open: boolean
     onClose(): void
-    onBind(): void
+    onBound(): void
```

Comment:
?

Reply:
this function is called after binding, bound is more suitable